src/status_im/contact/block.cljs

```diff
@@ -64,7 +66,13 @@
                        ;; add the contact to blocked contacts
                        (update :contacts/blocked (fnil conj #{}) public-key)
                        ;; update the contact in contacts list
-                       (assoc-in [:contacts/contacts public-key] contact))}
+                       (assoc-in [:contacts/contacts public-key] contact))
```

Comment:
I think we want the change in status-go, so that we don't have to ask the client to call this and works the same on desktop (https://github.com/status-im/status-go/blob/31feea9f15e06927db9d2ed8f0503043d477dc8f/protocol/messenger_contacts.go#L584 )